docs/reference/nuspec.md

```diff
@@ -74,7 +74,52 @@ A comma-separated list of the package creators using profile names on nuget.org.
 #### projectUrl
 A URL for the package's home page, often shown in UI displays as well as nuget.org. 
 #### licenseUrl
+> [!Important]
+> licenseUrl is being deprecated. Use license instead.
+
 A URL for the package's license, often shown in UI displays as well as nuget.org.
+#### license
+An SPDX license expression or path to a license file within the package, often shown in UI displays as well as nuget.org.
```

Comment:
This https://github.com/NuGet/docs.microsoft.com-nuget/blob/e8b1a32f0cdfb454d1aea2e5633bd08b64b91143/docs/reference/nuspec.md#example-nuspec-files
still mentions LicenseUrl

Reply:
yeah, that's fixed in my branch now.